test/cljs/snippets/manual.cljs

```diff
@@ -152,3 +152,21 @@
             (q/fill 0)
             (q/text-align :center :center)
             (q/text (str "Key events: " @counter) 250 250))))
+
+
+(q/defsketch mouse-and-key-pressed-variable
+  :size [500 500]
+  :draw (fn []
+          (q/background 240)
+          (q/fill 0 255 255)
+          (let [angle 0
```

Comment:
Can you add logic that writes text in the sketch saying that you should press mouse button and any key to test `mouse-pressed?` and `key-pressed?` Otherwise whoever runs this test won't have idea what it supposed to do. They'll just see empty white sketch.